docs/intro/vs/terraform.mdx

```diff
@@ -1 +1,26 @@
-TODO
+Terraform is an open-source Infrastructure as Code (IaC) software tool created by HashiCorp. With Terraform users define and provide data center infrastructure using a declarative configuration language known as HashiCorp Configuration Language (HCL).
+
+Terraform is a powerful toolset focused on cloud deployment. Nitric in comparison covers entires software development process, leveraging tools like Docker, Pulumi and others to enable mutli-cloud development.
```

Comment:
I think content like this sentence is going to be the most useful, I'd suggest adding more clarity and detail here. The Feature Comparison, for example, makes it feel like alternate tools for the same purpose, when that's not really the case. This page needs to help clarify that Nitric and Terraform are unrelated products in a similar space.

Reply:
Have provided more content to differentiate, not sure if it hits what you were after.